app/home.html

```diff
@@ -1,18 +1,9 @@
 <!doctype html>
 <html dir="ltr">
   <head>
-    <meta charset="utf-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no">
-    <title>MetaMask</title>
-    <link rel="preload" href="/_locales/en/messages.json" as="fetch" crossorigin="anonymous" />
-    <link rel="stylesheet" href="../ui/css/index.scss">
+    <%~ include("partial-head.html") %>
```

Comment:
I had to add a bunch of the same HTML to several html pages, so I used a "partial" for the `body`, and I thought I might as well use a partial for the head too. So its a bit out of scope of the PR, but I think it's worth it!